.ebextensions/nginx.config

```diff
@@ -332,31 +332,6 @@ files:
 
         charset utf-8;
 
-        # With Content Security Policy (CSP) enabled (and a browser that supports it (http://caniuse.com/#feat=contentsecuritypolicy),
-        # you can tell the browser that it can only download content from the domains you explicitly allow
-        # CSP can be quite difficult to configure, and cause real issues if you get it wrong
-        # There is website that helps you generate a policy here http://cspisawesome.com/
-        add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' https://dap.digitalgov.gov https://www.google-analytics.com; style-src 'self' 'unsafe-inline'; img-src 'self' data: https://www.google-analytics.com *.tile.openstreetmap.org; frame-ancestors 'none'; frame-src https://www.youtube.com;" always;
-
-        # MIME type sniffing security protection
-        #	There are very few edge cases where you wouldn't want this enabled.
-        add_header X-Content-Type-Options nosniff always;
-
-        # The X-Frame-Options header indicates whether a browser should be allowed
-        # to render a page within a frame or iframe.
-        add_header X-Frame-Options SAMEORIGIN always;
-
-        # Force the latest IE version
-        add_header X-UA-Compatible "IE=Edge";
-
-        # The X-XSS-Protection header is used by Internet Explorer version 8+
-        # The header instructs IE to enable its inbuilt anti-cross-site scripting filter.
-        add_header X-XSS-Protection "1; mode=block" always;
-
-        # HSTS (HTTP Strict Transport Security)
-        # This header tells browsers to cache the certificate for six months and to connect exclusively via HTTPS.
-        add_header Strict-Transport-Security "max-age=15768000" always;
```

Comment:
HSTS we should pass on all requests, but it's not the end of the world if we don't do it for the static content, especially if we're planning on preloading.